deploy/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -68,7 +68,7 @@ spec:
 {{- end }}
 {{- if .Values.discover.nodeAffinity }}
         - name: DISCOVER_AGENT_NODE_AFFINITY
-          value: {{ .Values.discover.nodeAffinity }}
+          value: {{ toYaml .Values.discover.nodeAffinity | quote }}
```

Comment:
why `quote` @hhk7734 ?

Reply:
Because it is an env vars, an error occurs if there is no `quote` when using dict type yaml.

Reply:
without `quote`

```yaml
        - name: DISCOVER_AGENT_NODE_AFFINITY
          value: |
            {{- toYaml .Values.discover.nodeAffinity | nindent 12 }}
```

Reply:
yes without quote make more sense to me, not sure how it is `nindent` if just specify the `quote` keyword, do you have any document that explains it

Reply:
https://helm.sh/docs/chart_template_guide/functions_and_pipelines/

Using `| quote` in `env[].value` is very common, so I cannot get the reason to replace to `nindent`.

Even you can see using `quote` above this line.

Reply:
thanks for the info!